packages/core/src/editor/types.ts

```diff
@@ -59,12 +60,23 @@ export type YooptaBlock = {
   create: (options?: CreateBlockOptions) => void;
   toggle: (options?: ToggleBlockOptions) => void;
   update: (id: string, data: Partial<YooptaBlockData>) => void;
+  delete: (options: DeleteBlockOptions) => void;
   updateElement: <TElementKeys extends string, TElementProps>(
     blockId: string,
     elementType: TElementKeys,
-    elementProps: TElementProps,
+    elementProps?: TElementProps,
   ) => void;
-  delete: (options: DeleteBlockOptions) => void;
+  createElement: <TElementKeys extends string, TElementProps>(
+    blockId: string,
+    elementType: TElementKeys,
+    elementProps?: TElementProps,
+    options?: CreateBlockElementOptions,
+  ) => void;
+  getElement: <TElementKeys extends string>(
+    blockId: string,
+    elementType: TElementKeys,
```

Comment:
Do we essentially need to know elementType to get it by id?

Reply:
changed